functions_and_scope_assignment.js

```diff
@@ -0,0 +1,287 @@
+// # Function Exercises
```

Comment:
Overall good functions that you created here. I think you understood functions pretty well and the fact that functions need to return values when needed. Try to resolve the last problem as it is not working as intended. Keep up the good work!